### PR TITLE
feat: add update attributes load test

### PR DESCRIPTION
**Pull request by gpoujol**

### CODE REVIEW

File: src/tests/update-attributes.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import { randomItem } from 'https://jslib.k6.io/k6-utils/1.2.0/index.js';
+
+export const options = {
+    vus: 10,
+    iterations: 100000,
+    thresholds: {
+        http_req_failed: ['rate<0.01'],
+        update_attributes_duration: ['avg<200', 'p(95)<400']
+    },
+    setupTimeout: '5m',
```
Comment (bobeal) [inline src/tests/update-attributes.js]: hmm... why?

File: src/tests/update-attributes.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+};
+
+const entities = new SharedArray('template entity', function () {
+    return JSON.parse(open('../data/template_entity.json')).entities;
+});
+
+const fragmentAttributes = new SharedArray('fragment update attributes', function () {
```
Comment (bobeal) [inline src/tests/update-attributes.js]: it is called "Attribute Fragment", not "Fragment Attribute" (in the latter case, fragment is read as a verb, this is definitely not the same thing)

File: src/tests/update-attributes.js (hunk 1 above)
Comment (gpoujol) [reply src/tests/update-attributes.js]: it was for my local test (I had done for 1000 entities, it took more time for the setup)

Comment (bobeal) [reply src/tests/update-attributes.js]: so it has to be removed? we don't want the test to fail if we create many entities in the setup, no?

Comment (gpoujol) [reply src/tests/update-attributes.js]: Yes I remove it

**Review by bobeal — APPROVED** [review]

**Pull request merged**